e2e/dapp-connections/WalletConnect.yaml

```
@@ -0,0 +1,73 @@
+appId: ${APP_ID}
+tags:
+  - browser
+  - parallel
+---
+- clearState
+- launchApp:
+    arguments:
+      isE2ETest: true
```

I think this test is failing on iOS because we need to clear the keychain to reset state properly. We've changed the default test setup recently to this.

```suggestion
- launchApp:
    clearState: true
    clearKeychain: true
    arguments:
      isE2ETest: true
```

This change can be applied to all new tests.